wire-ios-request-strategy/Sources/Object Syncs/Helpers/MLSMessage.swift

```diff
@@ -20,14 +20,31 @@ import Foundation
 
 /// A message that can be sent in an mls group.
 
-public protocol MLSMessage: OTREntity, MLSEncryptedPayloadGenerator {}
+public protocol MLSMessage: OTREntity, MLSEncryptedPayloadGenerator {
+
+    /// Messages can expire, e.g. if network conditions are too slow to send.
+    var shouldExpire: Bool { get }
+
+    /// Sets the expiration date with the default time interval and returns the date.
```

Comment:
```suggestion
    /// Sets the expiration date with the default time interval.
```

The current comment is a bit confusing because `setExpirationDate()` doesn't return a date.